warehouse/models/intermediate/transit_database/int_transit_database__urls_to_gtfs_datasets.sql

```diff
@@ -0,0 +1,16 @@
+WITH
+
+gtfs_datasets AS (
+    SELECT * FROM {{ ref('stg_transit_database__gtfs_datasets') }}
+),
+
+int_transit_database__urls_to_gtfs_datasets AS (
+    SELECT
+        base64_url,
+        airtable_record_id AS gtfs_dataset_key -- this is defined in {{ ref('dim_gtfs_datasets') }}
+    FROM gtfs_datasets
+    WHERE base64_url IS NOT NULL
+    QUALIFY ROW_NUMBER() OVER (PARTITION BY base64_url ORDER BY ts DESC) = 1
```

Comment:
I will add a comment but I want to make this as simple as possible for now. One potential option for the future would be to merge `gtfs_dataset` rows since our general assertion is that "GTFS dataset/feed" should be 1:1 with a URL.

Reply:
Yeah.... I mostly am just worried that this is a failure pattern that may not be obvious and I want to leave some trail so people could think to handle it.